code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,315 @@
+			usr << "	<b>Message:</b> [T.msg]"
+			usr << "	<b>Handling Admin:</b> [T.admin]"
+			usr << "	<b>Replied To:</b> [T.active]/<b><a href='?src=\ref[T];view_logs=\ref[T]'>(LOGS)</a></b>"
+			if(T.resolved == TICKET_UNRESOLVED)
+				usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Resolve)</a>"
+			else
+				usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Unresolve)</a>"
+
+		else
+			user << "Current Unresolved Ahelps:"
+			if(T.resolved == TICKET_UNRESOLVED)
+				usr << "<span class='adminnotice'><b><font color=red>#[T.id] By:</font> <A HREF='?priv_msg=[T.permckey];ahelp_reply=1'>[key_name(T.permuser)]</A></b><b> Ckey:</b> [T.permckey] <b>Name:</b> [T.permuser] <b>Unique ID:</b> [T.uID]</span>"
+				usr << "	<b>Controls:</b> [ADMIN_QUE(mob)] [ADMIN_PP(mob)] [ADMIN_VV(mob)] [ADMIN_SM(mob)] [ADMIN_FLW(mob)] [ADMIN_TP(mob)]"
+				usr << "	<b>Message:</b> [T.msg]"
+				usr << "	<b>Handling Admin:</b> [T.admin]"
+				usr << "	<b>Replied To:</b> [T.active]/<b><a href='?src=\ref[T];view_logs=\ref[T]'>(LOGS)</a></b>"
+				if(T.resolved == TICKET_UNRESOLVED)
+					usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Resolve)</a>"
+				else
+					usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Unresolve)</a>"
+
+/client/proc/ahelp_count(modifier)
+	var/amount
+	for(var/I in admintickets)
+		var/datum/adminticket/T = I
+		if(T.resolved == TICKET_UNRESOLVED)
+			amount++
+		if(T.resolved == TICKET_RESOLVED)
+			amount++
+	return amount
+
+
+/datum/adminticket/proc/listtickets()
+	set category = "Admin Help"
+	set name = "List Adminhelps"
+	set desc = "List all current adminhelps"
+
+	if(!check_rights(R_ADMIN))
+		src << "<font color='red'>Error: Only administrators may use this command.</font>"
+		return
+
+	var/count = 0
+
+	for(var/datum/adminticket/T in admintickets)
+		count++
+
+	usr << "<b>Current Ahelps:</b>"
+
+	if(count < 1)
+		usr << "	None"
+		return
+	for(var/I in admintickets)
+		var/datum/adminticket/T = I
+		usr << "<span class='adminnotice'><b><font color=red>#[T.id] By:</font> <A HREF='?priv_msg=[T.permckey];ahelp_reply=1'>[key_name(T.permuser)]</b></A><b> Ckey:</b> [T.permckey] <b>Name:</b> [T.permuser] <b>Unique ID:</b> [T.uID]</span>"
+		usr << "	<b>Controls:</b> [ADMIN_QUE(mob)] [ADMIN_PP(mob)] [ADMIN_VV(mob)] [ADMIN_SM(mob)] [ADMIN_FLW(mob)] [ADMIN_TP(mob)]"
+		usr << "	<b>Message:</b> [T.msg]"
+		usr << "	<b>Handling Admin:</b> [T.admin]"
+		usr << "	<b>Replied To:</b> [T.active]/<b><a href='?src=\ref[T];view_logs=\ref[T]'>(LOGS)</a></b>"
+		if(T.resolved == TICKET_UNRESOLVED)
+			usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Resolve)</a>"
+		else
+			usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Unresolve)</a>"
+
+/datum/adminticket/proc/listunresolvedtickets()
+	set category = "Admin Help"
```

Comment:
To be fair, it's also kinda nice, since the admin tab tends to get reeeeally cluttered if you have all the verbs showing.